packages/editor/src/editor-ui/editor-tooltip.tsx

```diff
@@ -19,14 +22,22 @@ export function EditorTooltip({
     isMac ? '⌘' : strings.keys.ctrl
   )
 
+  const positionClasses = {
+    top: 'bottom-full mb-2.5',
+    right: 'left-full top-1/3 transform -translate-y-1/3 translate-x-2.5',
+    left: 'right-full top-1/3 transform -translate-y-1/3 -translate-x-2.5',
+    bottom: 'top-full mt-2.5',
+  }
```

Comment:
I also don't mind the position approach in general but for those tooltips from a design perspective I'd prefer them being on the same vertical position. So basically I'd want them to be position top, but aligned `-left-6` `-right-6` for the sides. That would be a lot easier and cleaner with `className`,

Reply:
Pushed a change, it seems `-right-0` and `-left-0` look good, please let me know what you think.

Reply:
Thank you.
Personally I liked `-[direction]-6` better, that's why suggested it but I don't mind `[direction]-0`.
(but `-[direction]-0` hurt's a tiny bit 😉)